### DISCUSSION TITLE
CMake

### DISCUSSION THREAD

**Discussion by 4pastor** [nomacs/nomacs#1004] [Q&A] [answered]
Can anyone help with this problem installing Nomacs Image Viewer?  The CMake command always resulted in an error. Here's what's going on:
pastortheconstantgardener@Calvins-Air build % cd ~

pastortheconstantgardener@Calvins-Air ~ % rm -rf nomacs

pastortheconstantgardener@Calvins-Air ~ % git clone https://github.com/nomacs/nomacs.git

Cloning into 'nomacs'...
remote: Enumerating objects: 35925, done.
remote: Counting objects: 100% (911/911), done.
remote: Compressing objects: 100% (294/294), done.
remote: Total 35925 (delta 644), reused 843 (delta 604), pack-reused 35014
Receiving objects: 100% (35925/35925), 83.25 MiB | 333.00 KiB/s, done.
Resolving deltas: 100% (27532/27532), done.
pastortheconstantgardener@Calvins-Air ~ % cd nomacs

pastortheconstantgardener@Calvins-Air nomacs % mkdir build

pastortheconstantgardener@Calvins-Air nomacs % cd build

pastortheconstantgardener@Calvins-Air build % cmake ..

CMake Warning:
  Ignoring extra path from command line:

   ".."

CMake Error: The source directory "/Users/pastortheconstantgardener/nomacs" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
pastortheconstantgardener@Calvins-Air build % cd /Users/pastortheconstantgardener/nomacs
ls

3rd-party			Dockerfile			PULL_REQUEST_TEMPLATE.md	build				scripts
CODE_OF_CONDUCT.md		ImageLounge			README.md			crowdin.yml
CONTRIBUTING.md			LICENSE				appveyor.yml			installer
pastortheconstantgardener@Calvins-Air nomacs % cd /Users/pastortheconstantgardener/nomacs
rm -rf build

pastortheconstantgardener@Calvins-Air nomacs % mkdir build
cd build

pastortheconstantgardener@Calvins-Air build % cmake ..

CMake Warning:
  Ignoring extra path from command line:

   ".."

CMake Error: The source directory "/Users/pastortheconstantgardener/nomacs" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
pastortheconstantgardener@Calvins-Air build % 

My device info is:
macOS Somona 14.0

Thanks very much for helping!

**Comment by novomesk** [comment, marked as answer]
Try `cmake ../ImageLounge`
